extended-utxo-spec/extended-utxo-specification.tex

```diff
@@ -1121,15 +1133,19 @@ \section{Comments}
 occurs), but unspent outputs persist on-chain in the UTXO set until
 they are eventually spent.
 
-This strategy is not applied to data values since, as the name
-would suggest, we expect these to be small pieces of data (public
-keys, for example) which would not incur large storage charges.
+The same strategy applies to data values. However, it is useful to
+be able to record the full data value on the transaction that \emph{creates}
+an output. Hence the $\dataWits$ field \emph{may} contain mappings from
```

Comment:
Why _may_ and not _must_?  When this is eventually running on the blockchain, will some transactions contain this information and some not?  If so, who decides?

Reply:
We decided talking to Duncan that it should be optional. Not including it just means that it's harder for people to use your outputs, which might be fine if you're going to communicate the information via some off-chain channel.